### DISCUSSION TITLE
note/server

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#923] [Announcements]
# note/server

前情提要 关于防火墙的一些命令 提示 firewall-cmd（适用于 CentOS/RHEL 7 及以上） 通过上述命令开启 开启系统内部防火墙端口规则 80 443 22 21 Vim的常见命令 提示 在Linux环境下，Vim是一个极其强大的文本编辑器，它支持多种模式，包括命令模式、插入模式和可视模式等，在编辑过程中，我们经常需要清屏或者比较两个...

http://localhost:8083/note/server.html

**Comment by zhibilin** [comment]
Hello Word

**Reply by zhibilin** [reply]
Hello too
